dev.yml

```diff
@@ -13,7 +13,7 @@ up:
   - custom:
       name: Minikube Cluster
       met?: test $(minikube status | grep Running | wc -l) -ge 2 && $(minikube status | grep -q 'Configured')
-      meet: minikube start --kubernetes-version=v1.11.10 --vm-driver=hyperkit
+      meet: minikube start --kubernetes-version=v1.13.0 --vm-driver=hyperkit
```

Comment:
We can bump this to 1.15.xx (don't use 0 inevitably its missing some bug fix), but we'll need to bump our minimum supported version. https://github.com/Shopify/krane/pull/546 was the last time we bumped our min supported version.

Reply:
updated to 1.15.xx, but as far as i can tell the PR you linked already bumps the min supported version to 1.15-latest 🤔